web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Hi, 
I still really don't see the interest to break this rule, specially in that case (only two models). With your norm, we have to open the file to know which model is overloaded (or use rgrep, but the change of snake case to camel case for model is not finished I think). With the current OCA norm, it is not necessary.
The current https://github.com/OCA/maintainer-tools/pull/217 doesn't seem to allow this kind of design you are promoting.
I do not only discover code on my PC, but on github too (after odoo-code-search research) In that case, we have to open each model.py file to know what is inside. I'm not sure that github allows rgreping repository online...

That being said, that's not a "casus belli" for me, and I prefer to see your improvment in this OCA that not your improvment at all.

So, abstain.
Note : please, the choice of editor OCA developers usually use is really out of the scope of this PR...Otherwise, let's go talking about vim and emacs ! ;-)

Regards.